tests/Microsoft.Identity.Test.Unit/PublicApiTests/ClientCredentialWithRegionTests.cs

```diff
@@ -340,7 +340,7 @@ public async Task OtherCloudWithAuthorityValidationAsync()
 
                 var tokenHttpCallHandler = new MockHttpMessageHandler()
                 {
-                    ExpectedUrl = "https://eastus.login.windows-ppe.net/17b189bc-2b81-4ec5-aa51-3e628cbc931b/oauth2/v2.0/token",
+                    ExpectedUrl = "https://eastus.login.windows-ppe.org/17b189bc-2b81-4ec5-aa51-3e628cbc931b/oauth2/v2.0/token",
```

Comment:
That's quite subtle ... at least add a comment stating that `login.windows-ppe.org` is not known to MSAL.